src/curated-corpus/components/DuplicateProspectModal/DuplicateProspectModal.test.tsx

```diff
@@ -1,43 +1,75 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import { SnackbarProvider } from 'notistack';
 import { MockedProvider } from '@apollo/client/testing';
-import { AddProspectModal } from './DuplicateProspectModal';
-
-describe('The AddProspectModal component', () => {
-  const setApprovedItem = jest.fn;
-  const setCurrentProspect = jest.fn;
-  const setIsManualSubmission = jest.fn;
-  const setIsRecommendation = jest.fn;
-  const toggleApprovedItemModal = jest.fn;
-  const toggleModal = jest.fn;
-  const toggleScheduleItemModal = jest.fn;
-
-  it('should render the modal and the AddProspect form component', () => {
+import { SnackbarProvider } from 'notistack';
+import { DuplicateProspectModal } from './DuplicateProspectModal';
+import { getTestApprovedItem } from '../../helpers/approvedItem';
+import userEvent from '@testing-library/user-event';
+
+describe('The DuplicateProspectModal component', () => {
+  const testApprovedItem = getTestApprovedItem();
+  const toggleModal = jest.fn();
+  // mocking the window object's open method
+  const windowSpy = jest.spyOn(window, 'open');
+
+  it('should render the modal and all its elements ', () => {
+    render(
+      <MockedProvider>
+        <SnackbarProvider maxSnack={1}>
+          <DuplicateProspectModal
+            approvedItem={testApprovedItem}
+            isOpen={true}
+            toggleModal={toggleModal}
+          />
+        </SnackbarProvider>
+      </MockedProvider>
+    );
+
+    // fetch the modal by its heading and assert it rendered
+    expect(screen.getByText(/duplicate item/i)).toBeInTheDocument();
+
+    // fetch the item title in the modal and assert it rendered
+    // NOTE: I have to include the ":" before the title, otherwise it is unable to fetch
+    expect(screen.getByText(`: ${testApprovedItem.title}`)).toBeInTheDocument();
```

Comment:
This exact assertion for other modals has worked without adding the `:`. react testing library wasn't happy unless I did this 😕